packages/commonwealth/client/scripts/navigation/CommonDomainRoutes.tsx

```diff
@@ -111,6 +113,12 @@ const CommonDomainRoutes = ({
   contestEnabled,
   farcasterContestEnabled,
 }: RouteFeatureFlags) => [
+  <Route
```

Comment:
lets add this behind a feature flag (if this is for testing purposes)

Reply:
I will do this in v1.  The /editor URL isn't used anywhere but I can put it behind /_editor  or something if you want. It's just for test purposes.  

I wanted an easy way for Dillon or anyone on the team to play with and use the editor.  

That's part of the motivation behind v0.